src/main/java/mate/academy/springboot/swagger/service/ProductService.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.swagger.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.data.domain.PageRequest;
+
+public interface ProductService {
+    Product save(Product product);
+
+    Product getById(Long id);
+
+    void deleteById(Long id);
```

Comment:
```suggestion
    Product get(Long id);

    void delete(Long id);
```
by default everything goes by id